docs/change-log.md

```diff
@@ -2,6 +2,20 @@
 
 This document provides an overview over the changes we are making over time. Feel free to reach out to us in case of any specific questions.
 
+## Product Update February 5th, 2021
+
+- **New:** Added support to provision more than one resource of the same type for each workload.  
+![Multiple Resources of same Type](_assets/images/2021-02-05_Multiple_Resources.png)
```

Comment:
The attatched screenshot is confusing because it says "postgres" everywhere. I sent you an alternative screenshot in Slack.